src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -444,7 +446,7 @@ def _train_epoch(self, epoch: int, silent_mode: bool = False) -> tuple:
 
             # LOG LR THAT WILL BE USED IN CURRENT EPOCH AND AFTER FIRST WARMUP/LR_SCHEDULER UPDATE BEFORE WEIGHT UPDATE
             if not self.ddp_silent_mode and batch_idx == 0:
-                self._write_lrs(epoch)
+                self.epoch_start_lr_dict = self._get_lr_dict()
```

Comment:
What if one day we decide we want to log not only LR but also a WD? 
I'm suggesting to give a more common name to this container where one can put values to be logged by the end of epoch.

Reply:
nice one @BloodAxe

Reply:
I renamed like that, what do you guys think?
https://github.com/Deci-AI/super-gradients/blob/b175cb56639df38a5bdc90a7d4baf5fb0bed8965/src/super_gradients/training/sg_trainer/sg_trainer.py#L448